### PR TITLE
Fix broken images URLs for the README and add project metadata to the package.json file (to reflect it at the NPM website) 

### LINKED ISSUE #15: Fix missing images at the NPM website
We'll probably need to host the demo images somewhere in order for them to be correctly displayed
Screenshot_20220414_181255
.

### PR DESCRIPTION
**Pull request by smellyshovel**
Use GitHub raw URLs to for the README demo-images. Should close #15

### CODE REVIEW

**Comment by smellyshovel** [comment]
@wipfli Don't merge right away. I'll update the package version and we'll try to re-publish.

**Comment by smellyshovel** [comment]
Ready to merge and to trigger the publish workflow. I hope...

**Review by wipfli — APPROVED** [review]

**Pull request merged**
